tests/v2/Controller/ContactController/SwitchPrivateTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Tests\v2\Controller\ContactController;
+
+use App\DataFixtures\v2\BeneficiaryFixture;
+use App\DataFixtures\v2\MemberFixture;
+use App\Tests\Factory\BeneficiaireFactory;
+use App\Tests\Factory\ContactFactory;
+use App\Tests\v2\Controller\AbstractControllerTest;
+
+class SwitchPrivateTest extends AbstractControllerTest
+{
+    private const URL = '/contact/%s/toggle-visibility';
+
+    public function provideTestRoute(): ?\Generator
+    {
+        yield 'Should redirect to login when not authenticated' => [self::URL, 302, null, '/login', 'PATCH'];
+        yield 'Should redirect to list code authenticated as beneficiaire' => [self::URL, 302, BeneficiaryFixture::BENEFICIARY_MAIL, '/beneficiary/%s/contacts'];
```

Comment:
"Should redirect to list `code` authenticated as beneficiaire " --> petite faute